src/Core/Domain/Validation/RegexPattern.php

```diff
@@ -34,5 +34,7 @@ final class RegexPattern
     /**
      * Pattern for valid generic names
      */
-    const GENERIC_NAME = '/^[^<>={}]*$/u';
+    const PATTERN_GENERIC_NAME = '/^[^<>={}]*$/u';
+
+    const INVERSE_PATTERN_MESSAGE = '/[<>{}]/i';
```

Comment:
I don't understand the idea behind inverse pattern :thinking:

Reply:
basically it contains pattern which is valid when pregmatch is false. It is maybe possible to rewrite the pattern to fit normally :thinking:

Reply:
I think we could just write the inverted pattern, such as `/^[^\[<>{}\]]+$/i` in this case